pkgs/servers/web-apps/kavita/default.nix

```diff
@@ -26,6 +26,10 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     patches = [
       # The webroot is hardcoded as ./wwwroot
       ./change-webroot.diff
+      # Upstream removes database migrations between versions
+      # Restore them to avoid breaking on updates
+      # On update: check if more migrations need to be restored!
+      ./restore-migrations.diff
```

Comment:
Can we mention the version range "restored" for each patch? In this case, something like `0.7.2 -> 0.7.9`. This way we can easily know which patches to drop if we decide on a schedule.

Reply:
I added the range but adjusted the wording to make more clear that the v0.7.9 migrations themselves are not included in the patch.